LICENSE

```diff
@@ -187,7 +187,7 @@ APPENDIX: How to apply the Apache License to your work.
    same "printed page" as the copyright notice for easier
    identification within third-party archives.
 
-Copyright [yyyy] [name of copyright owner]
+Copyright 2014 Square, Inc.
```

Comment:
I wonder if this should actually be `Block. Inc.` now...

Reply:
Happy to approve the pull request once legal gets back to you 🙂

Reply:
😓

Reply:
(Let me dig in a bit, you've nerd sniped me...)

Reply:
Appreciate you buddy. Apologies for the landmine.